src/services/codefixes/parameterPropertyToDeclaration.ts

```diff
@@ -0,0 +1,90 @@
+/* @internal */

```

Comment:
Are these just helpers for the refactoring?  If so, why are they in the codefixes folder?

Reply:
It's cherry-picked by https://github.com/microsoft/TypeScript/pull/39669  
And `src\services\codefixes\generateAccessors.ts` also in the codefixes folder.

Reply:
Sorry, I'm not sure I understand.  If you're saying that other refactoring-specific helpers are in the codefixes folder, perhaps they should be moved as well?

Reply:
If some consumer in one of these PRs is in the codefixes folder, then I'm fine with their current location.  Otherwise, it seems strange to me.

Reply:
Emmm. Actually there's another quickfix references this in another pr. But I'm ok to put this into refactor or some other place.